.github/workflows/update-sponsors.yml

```diff
@@ -0,0 +1,19 @@
+name: Update Sponsors Data
+
+on:
+  branch: main
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  build:
+    name: Commit if needed
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: Codecademy/run-on-yarn@v1
+        with:
+          command: generate-sponsors
+      - uses: EndBug/add-and-commit@v7
+        with:
+          message: 'Update sponsors data'
```

Comment:
If we haven't yet, we'll need to set up some sort of environment variables for Git permissions...

Reply:
@JamesHenry was playing around with doing commits via actions for our weekly release job.
I remember he was having some trouble getting it to work.

Worst case scenario - we could make it could change it to weekly and make it put up a PR instead.